specs/signature-specification.md

```diff
@@ -177,29 +135,29 @@ Signed attributes/claims are additional metadata apart from the payload, which a
 - Claims which are informational and do not influence signature verification MUST NOT be marked critical.
 
-Notary v2 requires the signature envelope to support the following signed attributes/claims.
+Notary requires the signature envelope to support the following signed attributes/claims.
 
 #### Standard attributes
 
-- **Signing Scheme** (critical): A REQUIRED claim that defines the [Notary v2 Signing Scheme](./signing-scheme.md) used by the signature. This attribute dictates the rest of signature schema - the set of signed and unsigned attributes to be included in the signature. Supported values are `notary.x509` and `notary.x509.signingAuthority`.
+- **Signing Scheme** (critical): A REQUIRED claim that defines the [Notary Signing Scheme](./signing-scheme.md) used by the signature. This attribute dictates the rest of signature schema - the set of signed and unsigned attributes to be included in the signature. Supported values are `notary.x509` and `notary.x509.signingAuthority`.
 - **Signing Time**: A claim that indicates the time at which the signature was generated. Though this claim is signed by the signing key, it’s considered unauthenticated as a signer can modify local time and manipulate this claim. More details [here](#signing-time). This claim is REQUIRED and only valid when signing scheme is `notary.x509` .
 - **Authentic Signing Time** (critical): The authenticated time at which the signature was generated. This claim is REQUIRED and only valid when signing scheme is `notary.x509.signingAuthority` . More details [here](#signing-time).
 - **Expiry** (critical): An OPTIONAL claim that provides a “best by use” time for the artifact, as defined by the signer. More details [here](#expiry).
 - **Content Type** (critical): A REQUIRED claim that indicates the content type of the [payload](#payload). The supported value is `application/vnd.cncf.notary.payload.v1+json`. Other payload types MAY be supported in future.
 
 #### Extended attributes
 
-Implementations of Notary v2 signature spec MAY include additional signed attributes in the signature envelope.
+Implementations of Notary signature spec MAY include additional signed attributes in the signature envelope.
 These attributes MAY be marked critical, i.e. the attribute MUST be understood and processed by a verifier, unknown critical attributes MUST cause signature verification to fail.
 Usage of extended signed attributes which are marked critical in signature will have implications on portability of the signature, these are discussed in [Signature Portability](#signature-portability) section.
 
 #### Extended attributes for *Notation* Plugins
 
-This section documents extended attributes used by Notary v2 reference implementation *Notation* to support plugins.
+This section documents extended attributes used by Notary reference implementation *Notation* to support plugins.
```

Comment:
After you have set the context above, _Notary_ is not needed here. We can just stick with _Notation_. For example:

> This section documents extended attributes used by *Notation* to support plugins.